README.md

```diff
@@ -22,6 +22,12 @@
 - Twitter: [@isaacpitwa](https://twitter.com/isaacpitwa)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/isaac-pitwa)
 
+👤 **Ivan Gaona**
+
+- GitHub: [@ivaan1992](https://github.com/ivaan1992)
+- Twitter: [@ivangaona92](https://twitter.com/ivangaona92)
+- LinkedIn: [Ivan92](https://www.linkedin.com/in/ivan-linares-gaona/)
+
 
 ## 🤝 Contributing
```

Comment:
- [ ] I would recommend including all the steps to run the project locally. Always remember if a recruiter comes on this project he should have proper steps to see the project just by following the steps.

Hint:

```
$ git clone https://github.com/isaacpitwa/blogApp.git
$ cd blogApp
$ bundle
$ rails db:setup after setting the correct username and password in the database
$ rails s # To run the server

$ rspec # To test the application
```